src/path.h

```diff
@@ -3,15 +3,14 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
+#include <filesystem>
```

Comment:
Nit: if this header file is not using `filesystem` directly there's likely no reason to include it here. The include can be added to `path.cc` instead.

Reply:
👍